File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>
int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while ((n < 1) || (n > 8));
    for (int i = 0; i < n; i++) //loop for printing a new line
    {
        for (int j = n - i - 1; j > 0; j--) //loop for printing spaces
        {
            printf(" ");
        }
        for (int k = 0; k < i + 1; k++) //loop for printing hashes
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
from cs50 import get_int


def main():
    i = get_inclusive_int()
    for j in range(i):  # for loop to iterate over each line
        print((" " * (i - j - 1)) + ("#" * (j + 1)), end="")
        print()


def get_inclusive_int():  # fcn to get an integer from 1 to 8 inclusive
    while True:
        n = get_int("Height: ")
        if n >= 1 and n <= 8:
            break
    return n


main()

